src/components/ProductVersionMilestonesPage/ProductVersionMilestonesPage.tsx

```diff
@@ -39,6 +39,11 @@ export const ProductVersionMilestonesPage = ({ componentId = 'm1' }: IProductVer
             <ActionButton link="create">Create Milestone</ActionButton>
           </ProtectedComponent>
         </ToolbarItem>
+        <Text>
+          Product Milestone represents working release suffix of parent Product Version like <Label>0.CR1</Label> for example
+          Product Version of <Label>1.0.0.CR1</Label>. Each Product Version can contain multiple Product Milestones. Only one
+          Product Milestone could be set as active.
```

Comment:
`... the working ... the parent ... , for example` (the same is valid also for `ProductVersionReleasesPage.tsx`)

`... can be set ...`